packages/examples/cvat/exchange-oracle/src/handlers/escrow_cleanup.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import contextlib
+from typing import TYPE_CHECKING
+
+from cvat_sdk.api_client.exceptions import NotFoundException
+
+import src.cvat.api_calls as cvat_api
+import src.services.cloud as cloud_service
+from src.core.config import Config
+from src.core.storage import (
+    compose_data_bucket_prefix,
+    compose_results_bucket_prefix,
+)
+from src.services.cloud.utils import BucketAccessInfo
+
+if TYPE_CHECKING:
+    from src.models.cvat import Project
+
+
+def _cleanup_cvat(projects: list[Project]) -> None:
```

Comment:
It's possible to get some timeout or 5xx errors here, maybe it's better to try to remove all of the projects anyway.

Reply:
You mean remove files from cloud storage?

Reply:
Done. Errors will be ignored during deletion, but will be raised later in order to signal that something went wrong, so webhook processing can be retried later.